### PR TITLE
WAR-1576 : User details in CLI & HTML Result Summary

### PR DESCRIPTION
**Pull request by priyaarun23**
User Details has been added in CLI & HTML Result Summary.

### CODE REVIEW

**Comment by priyaarun23** [comment]
Verified the result with the user Karthik Ekanathan.

**Comment by brettleehari** [comment]
@priyaarun23,
I am Adding @keenan35i as a reviewer as he has recently changed the format of the result file nicely. Would be great if he looks into it.
Please add the screen shot before and after changes.

**Comment by priyaarun23** [comment]
before the fix
with_username

**Comment by priyaarun23** [comment]
User detail has been added at the left bottom. PFA.

File: warrior/WarriorCore/Classes/html_results_class.py [hunk 1]
```diff
@@ -158,20 +159,24 @@ def merge_html(self, dynamic_html):
...
-        return template_html[:index] + dynamic_html + template_html[index:] + self.get_war_version()
+        return template_html[:index] + dynamic_html + template_html[index:] + self.get_war_version() + self.get_user()
 
     def get_war_version(self):
         """ find the warrior version """
         path = self.get_path().split('warriorframework')[0] + 'warriorframework/version.txt'
         if os.path.isfile(path):
             version = open(path, 'r').read().splitlines()[1].split(':')[1]
             return '<div class="version">' + version + '</div>'
-        else:
```
Comment (brettleehari) [inline warrior/WarriorCore/Classes/html_results_class.py]: Good

File: warrior/WarriorCore/Classes/html_results_class.py [hunk 2]
```diff
@@ -158,20 +159,24 @@ def merge_html(self, dynamic_html):
...
-            return ''
+        return ''
+
+    def get_user(self):
+        """ find the user who executed the testcase """
+        user = getpass.getuser()
```
Comment (brettleehari) [inline warrior/WarriorCore/Classes/html_results_class.py]: Consider Try/catch block, while getting username

**Comment by priyaarun23** [comment]
@brettleehari 
Try/catch block is not needed as it takes the Linux/Windows Logged in user by default.
Checked in both Linux and Windows.

**Comment by priyaarun23** [comment]
@brettleehari Handled try/catch. Request you to review the same.

**Pull request merged**